### DISCUSSION TITLE
[Question] Have another binary for node version 8.x.x ?

### DISCUSSION THREAD

**Discussion by Chickyky** [uNetworking/uWebSockets.js#59] [Q&A] [answered]
my production use node version `8.14.0`
in source I see min version `uWebSockets.js` support is `uws_linux_64.node` equalivent min `versions.modules` is `64`.
Does `uWebSockets.js` support old version?

**Comment by ghost** [comment, marked as answer]
Nope

**Comment by Chickyky** [comment]
thanks for answer
